### ISSUE TITLE
possible to plot a single dimension?

### ISSUE THREAD

**Issue by studywolf** [nengo/nengo_gui#368]
can i plot a single represented dimension? would like to be able to if i can't!

**Comment by tbekolay** [comment]
Nope, not currently available. It would be nice indeed to be able to set indices on value plots, similarly to how we set indices on xy plots!
